tests/meta/report/test_comparison.py

```diff
@@ -0,0 +1,175 @@
+from datetime import datetime
+
+import pytest
+from tests.meta.report.comparison import (
+    is_float,
+    is_similar_values,
+    is_similar_lists,
+    value_exists_in,
+    dict_values_exists_in,
+)
+
+
+@pytest.mark.parametrize("data", [0, "0", 0.1, "0.1", "1.0", "10", True, False])
```

Comment:
Testing tests 😄 . I also do stuff like this all the time, but perhaps they do not need to be part of the CG test suite